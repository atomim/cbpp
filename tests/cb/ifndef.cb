#define MAGIC

#ifndef MAGIC
AddText "do not output me" 
#endif

#ifndef THIS_DOES_NOT_EXIST
AddText "output me pls"
#endif

AddText "this should be normal"
