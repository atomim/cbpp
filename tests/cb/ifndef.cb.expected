// #define MAGIC

// #ifndef MAGIC




AddText "output me pls"
// #endif

AddText "this should be normal"
